### PR TITLE
Send funds message update

### PR DESCRIPTION
**Pull request by Daniel-Wang**
## Priority
High

## Description
* Prevents the user from sending a 0 value payment
* Updates the toast to notify user of a failed 100% send failure (existing behaviour is a successful toast message)

## Screenshot

screenshot_20181108-133155

screenshot_20181108-121244

### CODE REVIEW

**Comment by Daniel-Wang** [comment]
Fixes #63

File: app/src/main/java/blockeq/com/stellarwallet/models/HorizonException.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package blockeq.com.stellarwallet.models
+
+import android.content.Context
+import blockeq.com.stellarwallet.R
+import blockeq.com.stellarwallet.utils.ErrorWrapper
+
+class HorizonException(private val transactionResultCode : String?,
```
Comment (dabitdev) [inline app/src/main/java/blockeq/com/stellarwallet/models/HorizonException.kt]: transactionResultCode and operationsResultCodes can be null?

**Comment by dabitdev** [comment]
it fixes #62 too
https://github.com/Block-Equity/stellar-android-wallet/issues/62

**Comment by Daniel-Wang** [comment]
Error message is now readable!

screenshot_20181112-125512

**Pull request merged**